docs/root/intro/arch_overview/http/http3.rst

```diff
@@ -5,16 +5,13 @@ HTTP3 overview
 
 .. warning::
 
-  HTTP/3 support is still in Alpha, and should be used with caution.
-  Outstanding issues required for HTTP/3 to go GA can be found
-  `here <https://github.com/envoyproxy/envoy/labels/quic-mvp>`_
-  For example QUIC does not currently support in-place filter chain updates, so users
-  requiring dynamic config reload for QUIC should wait until
-  `#13115 <https://github.com/envoyproxy/envoy/issues/13115>`_ has been addressed.
-
-  For general feature requests beyond production readiness, you can track
+  HTTP/3 downstream support is now GA, but continued improvements are coming, tracked in
   the `area-quic <https://github.com/envoyproxy/envoy/labels/area%2Fquic>`_ tag.
 
+  HTTP/3 upstream support is fine for in-datacenter use, but is not ready for internet use
```

Comment:
in-datacenter or intra-?

Reply:
I really mean "in your datacenters or on your network" and I don't know how to phrase that :-/  Ideas?

Reply:
How about something like "is fine for locally controlled networks, but is not ready for general internet use"?

Reply:
Should we use the terminology used in other places for the docs, similar to `The following extensions are intended to be hardened against untrusted downstreams but assume trusted upstreams`?

Reply:
This has nothing to do with trusted or untrusted, it has to do with if your routing hardware will do funky things to UDP packets, like allow N on a given tuple and then black hole.